core/src/main/java/com/oath/oak/OakRReference.java

```diff
@@ -124,12 +123,6 @@ public void unsafeCopyBufferToIntArray(int srcPosition, int[] dstArray, int coun
     }
 
     private ByteBuffer getTemporaryPerThreadByteBuffer() {
-        // need to check that the entire OakMap wasn't already closed with its memoryManager
```

Comment:
Please add a comment explaining the possibility of memory manager being closed and the fact that it will cause a NPE